src/settings-ui/Settings.UI/ViewModels/Flyout/AllAppsViewModel.cs

```diff
@@ -32,125 +32,269 @@ public AllAppsViewModel(ISettingsRepository<GeneralSettings> settingsRepository,
 
             var resourceLoader = Helpers.ResourceLoaderInstance.ResourceLoader;
             GpoRuleConfigured gpo;
-            if ((gpo = GPOWrapper.GetConfiguredAlwaysOnTopEnabledValue()) != GpoRuleConfigured.Disabled && gpo != GpoRuleConfigured.Enabled)
+            gpo = GPOWrapper.GetConfiguredAlwaysOnTopEnabledValue();
```

Comment:
This might go to be off-topic, but is there a different structure possible? I would think something like
```cs
gpo = GPOWrapper.GetConfiguredEnabledValue("AlwaysOnTop");
```
would make it just a bit more readable  and it is similar to the resourceloader structure.